src/picomsg/src/common_c_api/common.cpp

```diff
@@ -188,13 +188,21 @@ void pm_proto_addr_to_string_for_logs(const struct pm_proto_addr* const proto_ad
 
 void pm_proto_prefix_to_string_for_logs(const struct pm_proto_prefix* const proto_prefix,
 										char prefix_str[MAX_IP_PREFIX_STR_LEN]) {
-	if (! pm_proto_addr_to_string(&proto_prefix->prefix, prefix_str)) {
+	if (! pm_proto_prefix_to_string(proto_prefix, prefix_str)) {
 		strncpy(prefix_str, "(parse_error)", MAX_IP_PREFIX_STR_LEN);
 		prefix_str[MAX_IP_PREFIX_STR_LEN-1] = '\0';
-		return;
+	}
+}
+
+bool pm_proto_prefix_to_string(const struct pm_proto_prefix* const proto_prefix,
+							   char prefix_str[MAX_IP_PREFIX_STR_LEN]) {
+	if (! pm_proto_addr_to_string(&proto_prefix->prefix, prefix_str)) {
+		PICOMSG_LOG(PICOMSG_LOG_ERR, "parse error");
```

Comment:
```suggestion
		PICOMSG_LOG(PICOMSG_LOG_ERR, "failed to convert pm_proto_addr to string");
```
"parse error" is too general and doesn't mean anything when looking at logs